dev/Notifications/BadgeNotifications/BadgeNotification.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "BadgeNotification.h"
+#include <frameworkUdk/ToastNotificationsRT.h>
+
+using namespace Microsoft::Windows::BaseNotifications;
+
+namespace ToastABI
+{
+    using namespace ::ABI::Microsoft::Internal::ToastNotifications;
+}
+
+namespace Microsoft::Windows::BadgeNotifications {
+    // Default constructor
+    BadgeNotification::BadgeNotification() : BaseNotification()
```

Comment:
Can the `Base-` type just be merged into this type, since the API review eliminated this type being handed out?

Reply:
emoved baseNotificationManager but keeping BaseNotification because it simplifies the already existing Notification code. E.g - AppNotification now inherits BaseNotification and BadgeNotification also inherits BaseNotification. Otherwise duplicity of code would have happened. Also this makes both AppNotification and BadgeNotification extensible and easy to add new attributes